method-checklist.md

```diff
@@ -0,0 +1,32 @@
+# Method publishing checklist
+
+Please follow the checklist to ensure your content is ready for publishing on the [Methods Hub portal](https://methodshub.gesis.org/). *Note: the content submitted for publishing is held for review to crosscheck all requirements before publishing.*
+
+Methods that have [established communities](https://github.com/GESIS-Methods-Hub/guidelines-for-methods/blob/main/method-submission-guidelines.md#13alternative-for-established-methods) or are published by [trusted third parties](https://github.com/GESIS-Methods-Hub/guidelines-for-methods/blob/main/method-submission-guidelines.md#12trusted-third-party-review-bodies) can be considered as special case (bypassing some of the documentation requirements) for publishing through Methods Hub. *Please consult Methods Hub Team first (contact below) for further details.*
+
+| | Method publishing checklist|
+|--|------|
+| **Sec1**  | Preconditions: openness criteria (ref. [method preconditions](https://github.com/GESIS-Methods-Hub/guidelines-for-methods/blob/main/method-submission-guidelines.md#3-method-preconditions))|
+| ☐ | Developed in open-sourced programming language e.g., Python, R |
+| ☐ | Publicly accessible script |
+| ☐ | Open license for reuse | 
```

Comment:
Simplify the text. Use only "Open license". The term "for reuse" is too generic. For example, can you reuse "[Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International](https://creativecommons.org/licenses/by-nc-sa/4.0/deed.en)?